libbeat/_meta/fields.ecs.yml

```diff
@@ -336,43 +525,78 @@
       title: Event
       group: 2
       description: >
-        The event fields are used for context information about the data itself.
+        The event fields are used for context information about the log or metric event itself. A log is defined as an event containing details of something that happened. Log events must include the time at which the thing happened. Examples of log events include a process starting on a host, a network packet being sent from a source to a destination, or a network connection between a client and a server being initiated or closed. A metric is defined as an event containing one or more numerical or categorical measurements and the time at which the measurement was taken. Examples of metric events include memory pressure measured on a host, or vulnerabilities measured on a scanned host.
       type: group
       fields:
+    
         - name: id
+          level: core
           type: keyword
           description: >
             Unique ID to describe the event.
           example: 8a4f500d
-          phase: 1
+    
+        - name: kind
```

Comment:
this backport failed because `kind` is not present in 6.x: https://github.com/elastic/beats/pull/9962, do you plan to backport this PR?

Alternatively, should I copy back this file to 6.x? cc @ruflin

Reply:
The Packetbeat flow changes are not being back-ported to 6.x.